quickstart.py

```diff
@@ -261,14 +296,10 @@ def serversResponding(config):
       print "Timeout exceeded waiting for SymServer to start"
       return False
 
-    # Make sure we have a port value to contact SymServer on.
-    if 'configPath' in config:
-      SymServer.config.loadFile(config['configPath'])
-    elif 'configJSON' in config:
-      SymServer.config.loadJSON(config['configJSON'])
+    port = getSymServerConfig(config)['port']
```

Comment:
I was told to review everything with a secondary use case of pushing this to production. Antenna runs in production in a Docker container, thus I thought this was relevant.